tools/prologs-epilogs/README.md

```diff
@@ -0,0 +1,21 @@
+# Prologs and Epilogs
+
+The following scripts function simultaneously as both prolog and epilog scripts
+when used with the [external epilog and prolog][epe] feature of the
+slurm_controller_instance module.
+
+A typical approach is to stage these files to `/opt/apps/adm/slurm/scripts/` on
+the controller and then use symbolic links pointing from the directories that
+are iterated over by the `slurm_mux` external epilog and prolog feature.
+
+For example, if the following symbolic links are created:
+
+```
+/opt/apps/adm/slurm/prolog_slurmd.d/add-jobuser-to-sudoers.prolog_slurmd -> ../scripts/allow-jobuser-sudo
+/opt/apps/adm/slurm/epilog_slurmd.d/remove-jobuser-from-sudoers.epilog_slurmd -> ../scripts/allow-jobuser-sudo
+```
```

Comment:
I think the example would be more instructive if it made the example concrete. 

give the commands that would be called on the controller to set this up:
1. wget the script
2. simlink setup commands

Reply:
Done. I thought a simple script with explicit URLs and paths was better than defining a large # of variables. PTAL.